ansible/roles/docker/templates/alpine321.Dockerfile.j2

```diff
@@ -32,7 +33,7 @@ RUN apk add --no-cache --virtual .build-deps \
         ccache \
         openjdk21 \
         git \
-        procps \
+        procps-ng \
```

Comment:
According to <https://pkgs.alpinelinux.org/packages?name=procps*&branch=v3.21&repo=&arch=x86_64&origin=&flagged=&maintainer=>, `procps` doesn't exist anymore.